packages/xo-web/src/common/intl/messages.js

```diff
@@ -139,7 +139,7 @@ const messages = {
   homeHostPage: 'Hosts',
   homePoolPage: 'Pools',
   homeTemplatePage: 'Templates',
-  homeSrPage: 'Storages',
+  homeSrPage: 'Storage',
```

Comment:
Objects displayed on this page are *storage repositories*, do you think singular *storage* is clear enough?

Reply:
indeed in UK and US english the plural of storages / multiple storage items are referred to as `storage` - however `Storage Repositories` would be even clearer - can we fit this in there or would it make the selection box too large?

Reply:
*Storage repositories* is a bit long indeed, maybe we should use `SRs` like we do for `VMs`?

Any idea what XCP-ng center does?

Reply:
In Xencenter in the tabbed view they're just referred to as "Storage" which is pretty common. In the full list view where there's more room, it says "Storage Repositories". I think either Storage or SRs are our best bet

Reply:
![Screenshot 2021-04-09 180837](https://user-images.githubusercontent.com/1357246/114245613-cb329f00-995e-11eb-9322-4059b9817ed7.png)

Reply:
Ok, thanks :slightly_smiling_face: